server/config/topic_page_pb2.py

```diff
@@ -2,9 +2,10 @@
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
nit: can you check the protoc version with `protoc --version`. The current file is generated using 3.21.9 and I suspect this change is from using an older version.

Reply:
Thanks for pointing that out -- reverted for now.